docs/pouch_with_runV.md

```diff
@@ -0,0 +1,118 @@
+# Pouch with runV
+
+Container technology develops rapidly recently. It provides much convenience for application packing and resource utilization improvement. At the same time it brings benefits, LXC-based container technology also loses the appropriate security. Specifically, containers share operating system kernel on one machine. Once one container attemps to attack kernel, all workload on this host would be influenced.
+
+For some scenarios which are sensitive and strict to security, pure container technology has apparent defects. What's more, in cloud era multi-tenancy is the rigid demand from cloud customers. Therefore, strong isolation must be guaranteed seriously. In another word, container techonology needs more security hardening.
+
+[runV](https://github.com/hyperhq/runv) is a hypervisor-based runtime for [OCI](https://github.com/opencontainers/runtime-spec). It can easily helps improve host and container's security by virtualizing a guest kernel which isolates containers and host with a clearer boundary. Although hypervisor-base container brings more commitment on security, it loses some performance caused by virtualization technology. And it takes a litte more time to start a container, since more work needs done, like initrd.img loading, kernel loading, system process bootup and so on.
+
+## Architecture
+
+Supporting hypervisor-based OCI runtime is one of Pouch's goals. Pouch allows users to decide which kind of container to create. So with a unified entry of Pouch's API, users can create both hypervisor-based containers and LXC-based containers. With two kinds of carriers above, user's application can flexibly choose runtime on demand.
+
+Here is the architecture of Pouch's supporting both runV and runC:
+
+![pouch_with_runv_architecture](static_files/pouch_with_runv_architecture.png)
+
+## Quick Start
+
+Before installing, We should remind one important thing: **Pouch with runv can only work on PHYSICAL MACHINE**. Nested VMs currently are not supported yet. In addition, we should make sure that `containerd` and `pouchd` are already installed on the physical machine which is described in [INSTALLATION.md](../../INSTALLATION.md).
```

Comment:
hi, @allencloud i want to figure out why Pouch with runv can only work on physical machine, can you explain more detailed, or give me some advice where i can find the answers, thanks very much.

Reply:
Thanks for your question. Actually runv could be used in virtual machine. While I asked men in Hyper, their reply is that **runv could be used in nested VMs, while in poor performance and unstable **.

I could update this part to provide more details about the status.

Reply:
I tried on my virtual machine, but I failed. And I filed an issue in runv https://github.com/hyperhq/runv/issues/644.

Did you run runv in VM, or virtural machine? If you made it in VM, please tell me.
However, I still encourage trying physical machine.

Reply:
Yes, i run runv in mu VM, but we should use qemu with switch –enable-virtfs on.
Now, i just want to test the runv's function, so VM is good for me.